_data/authors.yml

```diff
@@ -0,0 +1,15 @@
+default_author:
+  name:           Your Name
+  picture:        your-photo.jpg
+  email:          your@email.com
+  # Use the coder's toolbox at http://coderstoolbox.net/string/#!encoding=xml&action=encode&charset=us_ascii to encode your description into XML string
+  description:    Some Details about yourself
+  # Social networking links used in footer. Update and remove as you like.
+  twitter:
+  facebook:
+  github:
+  linkedin:
+  instagram:
+  tumblr:
+  # For Google Authorship https://plus.google.com/authorship
+  google_plus:    "http://plus.google.com/123123123123132123"
```

Comment:
Should we add concrete details here before merging? Should this just be generic contact details for WeAllJS itself?